app/Search/Sapi3/Sapi3SearchService.php

```diff
@@ -34,6 +34,7 @@ public function findUiTPASOrganizers(string ...$ids): PagedCollection
             return new PagedCollection();
         }
 
+        //@todo I wonder if there is a bug here, because the function has a constraint on labels:uitpas, but this one does not?
```

Comment:
No, because this uses the provided IDs instead of searching an UiTPAS organizer on name.